packages/sonarwhal/src/lib/parsers/babel-config/babel-config.ts

```diff
@@ -0,0 +1,96 @@
+import * as path from 'path';
+import * as ajv from 'ajv';
+
+import { IScanEnd, IFetchEnd, Parser, BabelConfig, IBabelConfigInvalid, IBabelConfigParsed, IBabelConfigInvalidSchema } from '../../types';
+import { Sonarwhal } from '../../sonarwhal';
+import { loadJSONFile } from '../../utils/misc';
+
+
+export default class BabelConfigParser extends Parser {
+    private configFound: boolean = false;
+    private schema: any;
+
+    public constructor(sonarwhal: Sonarwhal) {
+        super(sonarwhal);
+
+
+        this.schema = loadJSONFile(path.join(__dirname, 'schema', 'babelConfigSchema.json'));
+        /**
+         * .babelrc => type: 'unknown' ('file-type' module doesn't support the type of 'json').
+         * babelrc.json => type: 'json' (file type from extention).
```

Comment:
We should also check the `package.json` file.

See: https://babeljs.io/docs/usage/babelrc/#use-via-packagejson